float64[] JointPosition
bool VisualiseRobot
---
int64 Collision
